frontend/src/components/Pages/FrontPage/RobotCards/PressureStatusView.tsx

```diff
@@ -13,16 +14,47 @@ const StyledTopography = styled(Typography)<{ $fontSize?: 24 | 16 | 18 | 32 | 40
 `
 export interface PressureStatusViewProps {
     pressure?: number
+    pressureStatus?: PressureStatus
```

Comment:
Why are all this optional? I think all should be mandatory

Reply:
Pressure is optional because not all robots have pressure. Could make it required but if so its necessary with changes such that only robots with pressure call this function.